src/Raven.Server/Web/System/StudioHandler.cs

```diff
@@ -123,7 +123,7 @@ private static FileInfo TryGetFileName(string basePath, string fileName)
             }
         }
 
-        [RavenAction("/studio/$", "GET", RequiredAuthorization = AuthorizationStatus.UnauthenticatedClients)]
+        [RavenAction("/studio/$", "GET", AuthorizationStatus.UnauthenticatedClients)]
```

Comment:
@ml054 Need to define what the minimum permissions we need just for error pages, and block the rest fro unauthenticated clients.